lang/es-MX.json

```diff
@@ -0,0 +1,1011 @@
+    "consent_manager": {
+        "data_collection_warning": "Utilizamos cookies (y otras tecnologías similares) para recopilar datos a fin de mejorar su experiencia de compra.",
+        "accept_all_cookies": "Aceptar todas las cookies",
+        "gdpr_settings": "Configuración",
+        "data_collection_preferences": "Preferencias de recopilación de datos de sitios web",
+        "manage_data_collection_preferences": "Aceptar las preferencias de recopilación de datos de sitios web",
+        "use_data_by_cookies": " utiliza datos recopilados por cookies y bibliotecas JavaScript para mejorar su experiencia de compra.",
+        "data_categories_table": "La siguiente tabla describe cómo usamos estos datos por categoría. Para excluirse de una categoría de recopilación de datos, seleccione «No» y guarde sus preferencias.",
+        "allow": "Permitir",
+        "accept": "aceptar",
+        "deny": "Denegar",
+        "dismiss": "Descartar",
+        "reject_all": "Rechazar todo",
+        "category": "{count} categoría",
+        "purpose": "Propósito",
+        "functional_category": "Funcional",
+        "functional_purpose": "Permite mejorar la funcionalidad, por ejemplo, vídeos y chat en directo. Si no se autorizan, es posible que algunas, o todas, estas características no funcionen correctamente.",
+        "analytics_category": "Análisis",
+        "analytics_purpose": "Brindar información estadística sobre el uso del sitio, por ejemplo, el análisis de la web, para poder mejorar el sitio web con el tiempo.",
+        "targeting_category": "Selección de clientes objetivo",
+        "advertising_category": "Publicidad",
+        "advertising_purpose": "Se usan para crear perfiles o personalizar el contenido para mejorar su experiencia de compra.",
+        "essential_category": "Características básicas",
+        "esential_purpose": "Son las esenciales para que funcionen el sitio y los servicios solicitados, pero no ejecutan ninguna funcionalidad adicional o secundaria.",
+        "yes": "Sí",
+        "no": "No",
+        "not_available": "N/A",
+        "cancel": "Cancelar",
+        "save": "Guardar",
+        "back_to_preferences": "Volver a las preferencias",
+        "close_without_changes": "Tiene cambios sin guardar en sus preferencias de recopilación de datos. ¿Está seguro de que desea cerrar sin guardar?",
+        "unsaved_changes": "Tiene cambios sin guardar",
+        "by_using": "Al utilizar nuestro sitio web, acepta",
+        "agree_on_data_collection": "Al utilizar nuestro sitio web, acepta la recopilación de datos tal y como se describe en ",
+        "change_preferences": "Puede cambiar sus preferencias en cualquier momento",
+        "cancel_dialog_title": "¿Está seguro de que desea cancelar?",
+        "privacy_policy": "Política de privacidad",
+        "allow_category_tracking": "Permitir el seguimiento de [CATEGORY_NAME]",
+        "disallow_category_tracking": "No permitir el seguimiento de [CATEGORY_NAME]"
+    }
+}
```

Comment:
Can you please add newline to the end of this and other files according to github suggestion?

Reply:
@yurytut1993 done!